src/Page.js

```diff
@@ -487,7 +487,8 @@ Page.prototype.insertSiteNav = function (pageData) {
     siteNavDataSelector('navigation').replaceWith(formattedSiteNav);
   }
   // Wrap sections
-  const wrappedSiteNav = `<div id="${SITE_NAV_ID}">\n${siteNavDataSelector.html()}\n</div>`;
+  const wrappedSiteNav = `<div class="${SITE_NAV_ID}" id="${SITE_NAV_ID}">`
```

Comment:
Although having the same name for IDs and classes works perfectly fine. We should avoid it as it is prone to confusion.

Since the class styles the `<a>` tag only, we can rename the class to `site-nav__link` ? 

[Reference to BEM naming convention](https://medium.freecodecamp.org/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849)

Reply:
`site-nav__link` would be directly applied to a `link` though.

https://seesparkbox.com/foundry/bem_by_example

Reply:
`site-nav__a`?

Reply:
I mean `site-nav__*` should go directly on the element.

The selector shouldn't be `.site-nav__a a { }`.

Reply:
I could change all generated links to follow `.site-nav__a`, but this probably means we should follow BEM in all our CSS files for consistency - is that what we want?

Alternatively, we could simply go with `.site-nav-link`/`.site-nav-a`.

Reply:
I'm open to either. @yamgent?

Reply:
> I could change all generated links to follow `.site-nav__a`,

Let's go with this (i.e. the BEM naming convention).